src/util/util.js

```diff
@@ -209,6 +212,39 @@ export function uuid(): string {
     return b();
 }
 
+/**
+ * Return whether a given value is a power of two
+ * @private
+ */
+export function isPowerOfTwo(value: number): boolean {
+    return Math.log2(value) % 1 === 0;
```

Comment:
`Math.log2` is [not available](https://caniuse.com/#feat=mdn-javascript_builtins_math_log2) in IE11, you have to use `Math.log(value) / Math.LN2`.

Reply:
Thanks, wasn't aware of that, updated!